generate_config.py

```diff
@@ -0,0 +1,91 @@
+import json
+import argparse
+import os
+import yaml
+import shutil
+from tableau_utilities.tableau_file.tableau_file import Datasource
+from tableau_utilities.tableau_server.tableau_server import TableauServer
+
+
+def do_args():
+    """ Parse arguments.
+
+    Returns: an argparse.Namespace
+    """
+
+    parser = argparse.ArgumentParser(description='List columns in Tableau datasources')
+    parser.add_argument(
+        '--server',
+        required=True,
+        help='Tableau Server URL. i.e. <server_address> in https://<server_address>.online.tableau.com',
+        default=None
+    )
+    parser.add_argument(
+        '--site',
+        required=True,
+        help='Site name. i.e. <site> in https://<server_address>.online.tableau.com/#/site/<site>',
+        default=None)
+    parser.add_argument('--api_version', help='Tableau API version', default='2.8')
+    parser.add_argument('--user', help='user name')
+    parser.add_argument('--password', help='password')
+    parser.add_argument('--token_secret', help='Personal Access Token Secret')
+    parser.add_argument('--token_name', help='Personal Access Token Name')
+    parser.add_argument('--datasource', help='The name of the datasources to generate a config for')
```

Comment:
```suggestion
    parser.add_argument('--datasource', help='The name of the datasources to generate a config for')
    parser.add_argument('--datasource_dir', help='The path to a directory containing datasources to generate a config for')
```
It would be nice to be able to create a config from directory of datasources we already have locally, as well.

Reply:
I've made the CLI work to create a config from a single file.  I don't think you will ever want to make things from a whole directly. You'll have an issue figuring out which is right when there are conflicts.  So I think you'll want to create a config 1 at a time, use MERGE to combine them. And look at diffs in github to manually resolve issues.  I've been adding configs to the master 1 datasource at a time in a PR and it's working well.